Create ui-c8y-1022-22-0-migrate-properties-selector-to-web-sdk-core.md (#3661)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1022-22-0-migrate-properties-selector-to-web-sdk-core.md

---
date: ""

> title: Asset properties selector component available in Web SDK for managing and displaying asset properties

Reviewer: ```suggestion
title: Asset properties selector component for managing and displaying asset properties available in the Web SDK 
```

> title: Asset properties selector component for managing and displaying asset properties available in the Web SDK 

Reviewer: ```suggestion
title: Asset properties selector component available in the Web SDK 
```

product_area: Application enablement & solutions
change_type:
  - value: change-QHu1GdukP
    label: Feature
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-61964
version: 1022.22.0
---

> The Asset Properties component is now available in the Web SDK core, providing a comprehensive interface for managing and displaying asset properties in a hierarchical tree structure. This component offers various customization options for both display and behavior, enabling a seamless experience when interacting with asset properties. It can already be used in the HTML widget to copy property paths that can be incorporated into templates and later used to display the matching property of the selected asset.

Reviewer: ```suggestion
The asset properties selector component is now available in the Web SDK, providing a comprehensive interface for managing and displaying asset properties in a hierarchical tree structure. This component offers various customization options for both display and behavior, enabling a seamless experience when interacting with asset properties. It can already be used in the HTML widget to copy property paths that can be incorporated into templates and later used to display the matching property of the selected asset.
```

---
date: ""

> title: Migrate properties selector to Web SDK core.

Reviewer 2: ```suggestion
title: Asset properties selector component available in Web SDK for managing and displaying asset properties
```
